notes.py

```diff
@@ -229,54 +234,80 @@ def parse_name(self, name: str) -> tuple[str, str, list[str], str]:
         return chord_note, quality, extensions, root
 
     def get_chord(
-            self, *, lower: 'Note' = Note('C', 0), raise_octave: dict[str, int] = None
+            self, *, lower: 'Note' = Note('C', 0), raise_octave: dict[int, int] = None
```

Comment:
This is changed back because now a chord can have repeated notes. Dict keys must be unique, so we make the keys the index of the note, rather than the note name.